src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_INDEX = 5;
+    private static final int MAX_UNITS = 100;
+    private static final Random random = new Random();

```

Comment:
```suggestion
    private final Random random = new Random();
```
Instances  shouldn’t  make static, and move it after static fields